install-deps.sh

```diff
@@ -306,6 +306,11 @@ else
     [ $WITH_ZBD ] && with_zbd=true || with_zbd=false
     [ $WITH_PMEM ] && with_pmem=true || with_pmem=false
     source /etc/os-release
+    # check if grafonnet-lib is there else download
+    if [ ! -d "grafonnet-lib-0.1.0" ];
+    then
+        $SUDO wget -c https://github.com/grafana/grafonnet-lib/archive/refs/tags/v0.1.0.tar.gz -O - | sudo tar -xz
```

Comment:
Why `$SUDO`?  If it's absolutely necessary, please `chown` the directory afterwards.

This PR is causing the Jenkins builders to fail a bunch of other jobs because the Jenkins user can't delete that directory.

e.g., https://jenkins.ceph.com/job/ceph-api/22209/console

Reply:
Updated the PR with this change. Sorry for the inconvenience @djgalloway .